cmd/gen-devnet/deployer.go

```diff
@@ -0,0 +1,130 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/cartesi/rollups-node/internal/config"
+	"github.com/cartesi/rollups-node/pkg/addresses"
+	"github.com/cartesi/rollups-node/pkg/ethutil"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	CONTRACT_OWNER_ADDRESS = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266"
+	SALT                   = "0x0000000000000000000000000000000000000000000000000000000000000000"
+)
+
+func deploy(ctx context.Context,
+	rollupsContractsPath string,
+	hash string) (DeploymentInfo, error) {
+	var depInfo DeploymentInfo
+
+	config.InfoLogger.Printf("deployer: deploying %v", rollupsContractsPath)
```

Comment:
@torives is changing the CLI command not to use the logger but instead just fmt.Printf. These commands don't need a full-blown logger, so I agreed with him in favor of making it simpler. Let's do the same here.

Reply:
Changes made at ee968dc7